src/library_async.js

```diff
@@ -383,14 +383,17 @@ mergeInto(LibraryManager.library, {
 
   emscripten_scan_registers: function(func) {
     Asyncify.handleSleep(function(wakeUp) {
```

Comment:
Might have been a bit easier to use `Asyncify.handleAsync` instead, which handles the callback automatically.